static/_redirects

```diff
@@ -374,6 +374,14 @@
 /docs/concepts/data-platform     /docs/data-platform   301!
 /docs/concepts/data-platform/*   /docs/data-platform/:splat 301!
 
+## Rename PDP About section
+
+/docs/data-platform/about-platform     /docs/data-platform/about   301!
+/docs/data-platform/about-platform/*   /docs/data-platform/about/:splat 301!
+
+/docs/data-platform/about-platform/platform-limits-and-status     /docs/data-platform/about/limits-and-status   301!
+/docs/data-platform/about-platform/platform-releases              /docs/data-platform/about/releases   301!
```

Comment:
These two should never become active, as line 380 will catch these and redirect to `/docs/data-platform/about/:splat`.

So probably you want:
```suggestion
/docs/data-platform/about/platform-limits-and-status     /docs/data-platform/about/limits-and-status   301!
/docs/data-platform/about/platform-releases              /docs/data-platform/about/releases   301!
```

Reply:
Thanks for catching this, Jan. That makes perfect sense.